meetings/2022-11-17.md

```diff
@@ -0,0 +1,128 @@
+* eP: Makes sense to me. I also don't think we can make it normative.
+* RG: I need to think about it and see if we need to add something normative for the server.
+* SC: We don't want to limit in what ways the protocol can be used. We could mandate closed list of information but that would be very limiting.
+* RG: For the client I'm not worrying about what it sends. For the server I see three buckets, what must be send and whay may be send. Te former we can specify what must not be send.
+* SC: I don't think we can specify what must not be send.
+* eP: Do you have some concrete example?
+* RG: For example exposing access control.
+* SC: It is possible that administrator or owner may want to track changes to access control.
+* RG: In the document I created I made a list of what would be the minimum information that has to be send. We could take a look at it.
+
+#### Add consider-subscription-resource-trust
+URL: https://github.com/solid/notifications/pull/118
+
+* SC: Client should be careful where it makes request. Especially a localhost, if I know that you have a server you your localhost, I could made you make request to your local server repeatedly.
+* RG: How do you stop this on the web. I can point to subscription on your server and clients would go make subscriptions there. Now that server would start getting subscription requests.
+* SC: There are legitimate reasons to have something on localhost. We just want to prevent clients blindly making requests. For example that client can have allow and deny lists of origins or particular resources. 
+* eP: I think we need separate considerations for clients to be restrained with their retries.
+* ...: I don't see clear threat with localhost so some reference there would be ok.
+* SC: This is mostly about making requests where you trust. In HTTP there is some consideration for redirects. Browser follows redirects automatically but for example curl needs a flag.
+* ...: I can think of some parental lock scenarios where warning might be needed.
+* eP: For me talking about trust here is too much, I don't see how implementations should put this consideration in pratctice.
+* SC: I recall Webmention recommends not sending them to localhost. https://www.w3.org/TR/webmention/#avoid-sending-webmentions-to-localhost
+* RG: I want to come back to CORS. In a manner similar to redirects the Access-Control-Allow-Origin header explicitly advises clients which domains it is allow to fetch data from. Perhaps a similar scheme is needed here?
+* SC: Different case I think. Client may not trust the subscription resource. Consider client running into a resource and the user wants to subscribe to updates e.g., updates from an intended political party, but then ends up subscribing to updates from an unwanted political party. This advisory is a warning so that client can factor in whatever they need to determine whether to trust the subscription resource or not before they send the subscription request.
```

Comment:
```suggestion
* SC: Different case I think. Client may not trust the subscription resource. Consider client running into a resource and the user wants to subscribe to updates, e.g., from an intended political party, but ends up subscribing to updates from an unwanted political party. This advisory is a warning so that client can factor in whatever they need to determine whether or not to trust the subscription resource before they send the subscription request.
```